phpseclib/File/X509.php

```diff
@@ -3794,7 +3794,7 @@ function &_subArray(&$root, $path, $create = false)
      * @param string $path optional absolute path with / as component separator
      * @param bool $create optional
      * @access private
-     * @return array ref or false
```

Comment:
How about using `&array` as the return type for reference to array? Not sure I've seen that used before.